tools/scripts/release/README.md

```diff
@@ -135,21 +145,13 @@ SKIP_OCKAM_BUMP=true ./tools/scripts/release/release.sh
 
 The release script can be called from any path.
 
-We also have a script to delete draft release, to delete draft
 
-```bash
-TAG_NAME=ockam_v0.71.0 ./delete_draft.sh
-```
+### Update A Draft Release
 
-Where TAG_NAME is the tag of the draft release.
+We can also update a draft release, to update a draft release
```

Comment:
What will be retained in the draft release?

Reply:
Updated, can you review?